heroku.yml

```diff
@@ -0,0 +1,6 @@
+build:
```

Comment:
(If you do switch to pushing with the Heroku CLI via the `script` deploy mentioned above, you should find you don't need this file)